com.unity.multiplayer.mlapi/Runtime/Messaging/MessageBatcher.cs

```diff
@@ -29,13 +30,23 @@ public SendStream()
         // Stores the stream of batched RPC to send to each client, by ClientId
         private readonly Dictionary<ulong, SendStream> m_SendDict = new Dictionary<ulong, SendStream>();
 
+        public void Shutdown()
+        {
+            foreach (var kvp in m_SendDict)
+            {
+                kvp.Value.Writer.Dispose();
+                kvp.Value.Buffer.Dispose();
+            }
+            m_SendDict.Clear();
+        }
+
         // Used to store targets, internally
         private ulong[] m_TargetList = new ulong[0];
 
         // Used to mark longer lengths. Works because we can't have zero-sized messages
         private const byte k_LongLenMarker = 0;
 
-        private void PushLength(int length, ref PooledNetworkWriter writer)
+        internal static void PushLength(int length, ref PooledNetworkWriter writer)
```

Comment:
This looks oddly familiar to `WriteIntPacked`, is there a reason why we have a custom function for writing the length instead of using the default packed write?

Reply:
No idea! Maybe it's for efficiency since it knows the value will be either one or two bytes specifically?

Reply:
@TwoTenPvP can you remember?